src/algorithms/PVT/gnuradio_blocks/rtklib_pvt_gs.cc

```diff
@@ -511,11 +518,14 @@ rtklib_pvt_gs::rtklib_pvt_gs(uint32_t nchannels,
     d_beidou_dnav_almanac_sptr_type_hash_code = typeid(std::shared_ptr<Beidou_Dnav_Almanac>).hash_code();
 
     d_start = std::chrono::system_clock::now();
+
+    d_spoofing_detector = spoofing_detector;
 }
 
 
 rtklib_pvt_gs::~rtklib_pvt_gs()
 {
+    std::cout << "\nTotal iterations: " << d_total_pvt_measurements << "\n";
```

Comment:
This kind of `std::cout` output is uninformative. There is no context to know what "Total iterations" relates to. A LOG message would be better.

But if you **must** use `std::cout`, flush the stream so the output is more-or-less in time order

Reply:
Noted, it was just for debugging, I have switched to dlog now.